app/models/server_puppet_data.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class ServerPuppetData
+  attr_reader :client
+
+  def initialize
+    @client = PuppetdbClient.new(Settings.PUPPETDB_ENDPOINT)
+  end
+
+  def self.run
+    new.load_data
+  end
+
+  def load_data
+    Server.find_each do |server|
```

Comment:
Ah, yep, you're right. However, the default batch when not listed in the `find_each` block is 1000 records. We currently don't have, and hopefully will never have that many servers to manage. Should we set that batch size if concerned about efficiency?